sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -125,7 +129,7 @@ case class DescribeFunctionCommand(
 
           if (isExtended) {
             result :+
-              Row(s"Extended Usage:\n${replaceFunctionName(info.getExtended, info.getName)}")
+              Row(s"Extended Usage:${replaceFunctionName(info.getExtended, info.getName)}")
```

Comment:
That makes sense in a way but I meant adding `stripMargin` here might virtually mean allowing another style document such as something like the one below..

```scala
@ExpressionDescription(
  usage = "_FUNC_(expr) - Returns the mean calculated from values of a group.",
  extended =
    """|
       |    Arguments:
       |      expr - a numeric expression.
       |  """)
```

and ..

```scala
@ExpressionDescription(
  usage = "_FUNC_(expr) - Returns the mean calculated from values of a group.",
  extended = """
    Arguments:
      expr - a numeric expression.
  """)
```

I guess if I am supposed to fix the doc without knowing this will execute actually `stripMargin`, I think I would get confused while fixing it.

I would like to leave as it is rather than adding some sugar and magic on this but force the latter style because I guess currently that forced style satisfies all cases.

(Of course no strong preference but I just wanted to elaborate what I meant to say.)